content/about/brand.md

```diff
@@ -9,23 +9,22 @@ layout: "brand"
 <div class="container">
 <div class="row mt-30">
 <div class="col-lg-6 text-center panel">
-<img src="../../images/logos/grasslogo.svg" width="33%" alt="GRASS GIS">
+<img src="../../images/logos/gs-green.svg" width="33%" alt="GRASS">
```

Comment:
Let's call the files grass... instead of gs...

Reply:
I've generally been using gs for the GRASS assets namespace, logos, css variables. Is this just a personal preference or do you have another reason you want to rename it?

Reply:
These files will be downloaded by people, so they end up with "gs something" files possibly without context. Also, grass is not a long word for a filename. CSS property prefix or Python import alias are different because they have limited scope and are repeated often (I'm not sure about the other uses of gs). I do prefer explicit rather than implicit in naming.

Reply:
Good point. I'll update them.